expander/ds/markdown.ftd

```diff
@@ -0,0 +1,349 @@
+-- import: fastn/processors as pr
+-- import: fastn.com/expander/lib
+
+-- my-ds.page: markdown in `doc-site`
+
+-- ds.markdown:
+if: { source == "default" || source == "build" }
+
+In this video we will see how to use markdown in `doc-site`.
+
+/-- ds.youtube:
+if: { source == "default" || source == "build" }
+v: 
+
+-- lib.video-audience: How to use markdown in `doc-site`
+if: { source == "planning" }
+owner: Ajit
+aud: Website Builders
+
+Gives the idea about the markdown syntax allowed in `fastn`
+
+-- ds.h1: Straight to the point
+if: { source == "planning" }
+
+By importing the `doc-site` package and using the `ds.markdown` component, you
+gain access to a powerful tool for creating documentation and content in
+`fastn` projects.
+
+The `ds.markdown` component not only supports the basic Markdown
+syntax provided by the `ftd.text` kernel component but also offers additional
+properties such as `color` and `role`.
+
+
+-- ds.code:
+lang: ftd
+
+\-- component markdown:
+body body:
+optional boolean collapsed:
+optional caption title:
+optional boolean two_columns:
+optional string id:
+
+\-- ftd.column:
+width: fill-container
+id if { $markdown.id != NULL }: $markdown.id
+
+\-- ftd.text: 
+text: $markdown.body
+role: $inherited.types.copy-regular
+color: $inherited.colors.text
+classes: markdown
```

Comment:
```suggestion
```

`classes: markdown` is not needed.

Reply:
I took this code from `tf.markdown` component of the typography package in fastn-community.